### PR TITLE
edit command to create config file

### PR DESCRIPTION
**Pull request by Oziomajnr**
"touch" is not a windows or power shell command

> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7293)

### CODE REVIEW

**Comment by Oziomajnr** [comment]
I have signed the Contributor License Agreement (CLA)

**Comment by tengqm** [comment]
/unassign @tengqm

Not familiar with PowerShell.

File: docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ re-run install-kubectl.ps1 to install latest binaries
        cd C:\users\yourusername (Or wherever your %HOME% directory is)
        mkdir .kube
        cd .kube
-       touch config
+       New-Item config -type file
```
Comment (heckj) [inline docs/tasks/tools/install-kubectl.md]: I'm not super-confident in this space, but it looks like this should be:

    New-Item -ItemType file config

per the Q&A at https://superuser.com/a/540935/43185

Comment (Oziomajnr) [reply docs/tasks/tools/install-kubectl.md]: image
they are both equivalent commands, I created the two files above using both commands

**Pull request merged**